src/test/java/JSONParserTest.java

```diff
@@ -0,0 +1,75 @@
+import Group15.Api.Exercise;
+import Group15.JSONParser;
+import Group15.Workout;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class JSONParserTest
+{
+    // Can hold objects between tests:
+    private String exercisesFilePath;
+    private String workoutsFilePath;
+    private List<Exercise> exercises;
+    private List<Workout> workouts;
+
+    private void initExercises()
+    {
+        exercises = new ArrayList<>();
+
+        Exercise pushup = new Exercise("Pushup", "Push your body up and down", "Strength", "Chest", "None", "Easy", "None");
```

Comment:
Can the JSONParser handle mutliple bodyparts?

Reply:
Yes